ui/lib/sync/addon/templates/secrets/overview.hbs

```diff
@@ -7,4 +7,5 @@
   @destinations={{this.model.destinations}}
   @totalVaultSecrets={{this.model.associations.total_secrets}}
   @featureEnabled={{this.model.featureEnabled}}
+  @adapterError={{this.model.adapterError}}
```

Comment:
Instead of passing the adapter error all the way down from `this.model` could we instead just check for `this.model.featureEnabled.isAdapterError` here? That would clean up the logic a bit in the route because we wouldn't have to track it all the way from the route. 

I like it being an arg here to the component, because that will make testing much easier!

Reply:
Brilliant, thank you. this was the brain cell(s) I did not have today.